README.md

```diff
@@ -124,12 +179,18 @@ for licenses considered forbidden.
 
 ## Usages
 
-Report usage:
+Report usage (csv output):
 
 ```shell
 go-licenses csv <package> [package...]
 ```
 
+Report usage (using custom template file):
+
+```shell
+go-licenses template <package> [package...] --template=<template_file>
```

Comment:
"report" sounds good.  If the template argument is optional, what would the default behavior be if not provided?  We could have a default template that does CSV output, but I'm not sure if we could easily replicate the [quoting behavior](https://cs.opensource.google/go/go/+/refs/tags/go1.19:src/encoding/csv/writer.go;l=157) from the encoding/csv package.  what do you think?

Reply:
Well, many of the fields that require quoting shouldn't ever appear in the type of data we're handling... quotes and newlines would get escaped anyway.  commas can't appear in go package names, and shouldn't ever appear in SPDX identifiers, but I believe they're valid in URLs.  At least, they don't seem to get [escaped by default](https://go.dev/play/p/FsOIh4kRENS).  Maybe the default behavior is CSV where all fields are quoted, or at least the license URL is?

Reply:
or I guess we don't even try to replicate encoding/csv with a template, and just branch the logic so that if a template is not specified, we use the real encoding/csv package like you do with the csv subcommand today?

Reply:
Yes, just branch the logic makes the most sense.
Ideally, it can be a shared function between new and old command.

Reply:
👍🏻 that's exactly what I ended up going with.